pkg/storage/testing/types.generated.go.tmp

```diff
@@ -0,0 +1,365 @@
+/*
+Copyright 2015 The Kubernetes Authors All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
```

Comment:
This might be the cause of the test failure and, anyway, we shouldn't commit these tmp files. Please remove it from the PR.